docs/native/src/testing/testcases.rst

```diff
@@ -1,3 +1,163 @@
+     * Please note the version of the Office app you tested against. This information can be found under :menuselection:`Settings --> [App] --> Version`
+	 
+1 - Open from Company App - First Install
+-----------------------------------------
+*This test verifies the flow of using Office for the first time from the Company app. Repeat for each supported file type and each Office app.*
+
+#. Start with a fresh install of Company app. Ensure Office is not installed.
+#. Boot up company app and login.
+ **RESULT: General promotion for Office should be shown the first time after upgrading the company app.**
+
+
+#. Browse to a supported file type
+ **RESULT: "Open with Microsoft [app]" promotion, drawing attention to control and enabling open with Office once per first open for each supported file type. "Open with Microsoft should be top choice if multiple choices are available. If a list is not shown Office should be the default app for opening the file.**
+
+#. Activate control to open in Office
+
+**RESULT: User should be sent to app store page for the corresponding app.**
+
+#. Install the Office app.
+#. After installation, go back to the Company app and activate the control to open in Office again.
+
+**RESULT: Office should start. User should be prompted for credentials to Company Service.**
+
+#. Enter credentials for company service.
+
+**RESULT: File should open in read only mode.**
+
+#. Click :guilabel:`Sign In` and sign in with a free Microsoft Account.
+
+**RESULT: File should open in edit mode if user is a consumer and read-only mode if the user is a commercial user.**
+
+#. Make changes [you will need to sign in with a subscription account for testing commercial user]
+#. Click :guilabel:`Back (<-)`
+#. Click :guilabel:`Open`
+
+**RESULT: Confirm company service is shown as a place.**
+
+2 - Open from Office - Fresh Install
+------------------------------------
+*This test verifies the flow of using Company Service for the first time from Office.*
+
+#. Launch a fresh install of Office.
+#. Go through the First Run Experience.
+#. Skip :guilabel:`Sign In.`
+#. Go to :menuselection:`Open --> Add a Place`
+
+**RESULT: Company service shows up. Verify the name and icon of your service.**
+
+#. Select your Company Service.
+#. Enter credentials.
+
+**RESULT: Root folder should show.**
+
+#. Browse around the folder structure in your service.
+
+**RESULT: Browse works as expected.**
+
+#. Open a file from :guilabel:`Browse`.
+
+**RESULT: File should open in read-only mode.**
+
+#. Click :guilabel:`Sign In` and sign in with a free Microsoft Account.
+
+**RESULT: File should open in edit mode if user is a consumer and read-only mode if the user is a commercial user.**
+
+#. Make changes [you will need to sign in with a subscription account for testing commercial user]
+#. Click :guilabel:`Back (<-)`
+#. Click "Open"
```

Comment:
This is a great place for `guilabel`. We can clean these up as we find them, though.